snprc_scheduler/src/org/labkey/snprc_scheduler/SNPRC_schedulerController.java

```diff
@@ -233,9 +238,17 @@ else if (parentObjectId != null )
                         // add filters to return all research projects
                         filters.add(new SimpleFilter(FieldKey.fromParts("ProjectType","Description"), "Research", CompareType.EQUAL));
                     }
+
+                    if (eventDateString != null) {
+                        try {
+                            eventDate = DateUtil.parseDateTime(eventDateString, Timeline.TIMELINE_DATE_FORMAT);
+                        } catch (ParseException pe) {
+                            throw new ValidationException("EventDate must be a valid Date");
+                        }
+                    }
                 }
 
-                List<Map<String, Object>> projects = SNDService.get().getActiveProjects(getContainer(), getUser(), filters, true);
+                List<Map<String, Object>> projects = SNDService.get().getActiveProjects(getContainer(), getUser(), filters, true, eventDate);
```

Comment:
It appears we are missing an SND update for this method signature.

Reply:
I will open the PR for the SND changes as well